gnosis/safe-react
# Replace icons and styles on Settings and Address Book

## What it solves
Closes #2347 
Description: We must use the new "edit" and "delete" icons in all the app.

## How this PR fixes it
Replace icons on Settings: 
1. [Remove Safe](https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5f5b9526f76cfd73a80f8299)

2. [Manage Owners](https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5f5b952c6cbbc545446dfd03)
(I kept the icons aligned to the right as in the current implementation to keep it consistent across the app. Eg. [Address Book](https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5f5b904b0faf6c76b20961f3))

3. [Spending Limit](https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5e7cafe915e11f8afe4d5dbf) 

4. Advanced (no design, but is similar to Spending Limit)

Also 2 icons were replaced on [Create new Safe screen ](https://app.zeplin.io/project/5b83b4fa75f90943c86ed14f/screen/5f5b79b428e3e577242217e5)

## How to test it
Steps:
In Settings: 
- Check at the top-right of the screen the Remove Safe "Delete" icon. 
- Open Manage Safe owners list as an owner and check "Edit" and "Delete" icons for the entities in the table.
- Open Spending limit list and check "Delete" icon.
- Open Advanced list and check and "Delete" icon.
 
Create new Safe:
- Fill the name input of the new safe. In the second step, click on "add another Owner". Check the "Qr" and "Delete" icons.
